content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -319,23 +319,17 @@ kubectl apply -f https://docs.projectcalico.org/v3.8/manifests/canal.yaml
 {{% /tab %}}
 
 {{% tab name="Cilium" %}}
-For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/kubernetes/).
-
 For Cilium to work correctly, you must pass `--pod-network-cidr=10.217.0.0/16` to `kubeadm init`.
 
-These commands will deploy Cilium with its own etcd managed by etcd operator.
-
-_Note_: If you are running kubeadm in a single node please untaint it so that
-etcd-operator pods can be scheduled in the control-plane node.
-
-```shell
-kubectl taint nodes <node-name> node-role.kubernetes.io/master:NoSchedule-
-```
+Cilium can provide kube-proxy functionality. To disable kube-proxy, pass
+`--skip-phases=addon/kube-proxy` to `kubeadm init`. Afterwards, you will need
+to configure Cilium to enable the BPF NodePort feature and to specify the kube-apiserver
+address, see [Kubernetes without kube-proxy](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/).
```

Comment:
i would change this whole paragraph to the following:

> Cilium can be used as a replacement for kube-proxy, see [Kubernetes without kube-proxy](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/).

and move it right above:

> For more information about using Cilium with Kubernetes, see [Kubernetes Install guide for Cilium](https://docs.cilium.io/en/stable/kubernetes/).

my rationale is that we don't want to first class this yet:
- it's Beta as per the Cilium docs.
- i haven't heard comments from SIG Network about it.

Reply:
@neolit123 Looks good to me as well. The [Kubernetes without kube-proxy](https://docs.cilium.io/en/stable/gettingstarted/kubeproxy-free/) already contains the information to how run kubeadm without kube-proxy so there's no point in duplicating it here.

Reply:
Thanks for the review!